custom/icds/models.py

```diff
@@ -0,0 +1,164 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import uuid
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.functional import cached_property
+from django.utils.translation import ugettext_lazy as _
+from django.db.models.signals import pre_delete
+
+from corehq.apps.app_manager.dbaccessors import get_build_doc_by_version
+from custom.icds.const import (
+    FILE_TYPE_CHOICE_ZIP,
+    FILE_TYPE_CHOICE_DOC,
+    DISPLAY_CHOICE_LIST,
+    DISPLAY_CHOICE_FOOTER,
+)
+from custom.icds.utils.ccz_hosting import CCZHostingUtility
+from custom.icds.validators import (
+    LowercaseAlphanumericValidator,
+)
+from custom.nic_compliance.utils import hash_password
+
+
+class CCZHostingLink(models.Model):
+    identifier = models.CharField(null=False, unique=True, max_length=255, db_index=True,
+                                  validators=[LowercaseAlphanumericValidator])
+    username = models.CharField(null=False, max_length=255)
+    # b64_aes_encrypt'ed raw password is stored in DB
```

Comment:
Is this comment still relevant? According to https://github.com/dimagi/commcare-hq/pull/24129/files#r283154584 it is now salted and hashed.

Reply:
It's not. removed.
Thanks